app/api/routes.py

```diff
@@ -39,6 +39,15 @@ def categories():
 
 
 # Helpers
+def standardize_response(data):
```

Comment:
Possibly have this function take in a status and response_data ("data" or "error" and then the formatted data so we don't repeat our-self in error handling?

Reply:
If I do that, where should that function live that it would be accessible to both `handlers.py` as well as `routes.py`?

Reply:
maybe in utils like the paginator?